docs/kbs.yaml

```diff
@@ -259,6 +270,25 @@ components:
           description: >-
             Encryption algorithm used to encrypt the response payload.
 
+    AttestationPolicy:
+      required:
+        - type
+        - policy_id
+        - policy
+      properties:
+        type:
+          type: string
+          description: >-
+            Policy Engine type, for example: "opa"
+        policy_id:
```

Comment:
So the id here is used to index possibly multiple policy entries in the `policy` payload below?

Reply:
The ID here is the ID of the policy payload below. Once setting operation can only set a single policy, but there may be multiple policies in the policy engine at the same time. Different policies can be selected according to the situation during each evaluation, as mentioned here: https://github.com/confidential-containers/attestation-service/pull/92#issuecomment-1541543936

Reply:
Ah, I did not see you changed the AS API. That makes sense to me, this is really forwarding the verifier policy to the AS, and maps the policy to an ID.